test/Swashbuckle.AspNetCore.SwaggerGen.Test/Fixtures/FakeController.cs

```diff
@@ -54,6 +54,15 @@ public void ActionWithIntParameterWithRequiredAttribute([Required]int param)
         public void ActionWithIntParameterWithSwaggerIgnoreAttribute([SwaggerIgnore] int param)
         { }
 
+        public void ActionWithAcceptFromHeaderParameter([FromHeader] string accept)
+        { }
+
+        public void ActionWithContentTypeFromHeaderParameter([FromHeader(Name = "Content-Type")] string contentType)
+        { }
+
+        public void ActionWithAuthorizationFromHeaderParameter([FromHeader] string authorization)
+        { }
```

Comment:
Add an extra parameter that's not one of these so we're checking we're not over-filtering the headers?